sdk/eventhub/Azure.Messaging.EventHubs/samples/Program.cs

```diff
@@ -251,6 +539,21 @@ private static CommandLineArguments ParseArguments(string[] args)
               .Select(type => (IEventHubsSample)Activator.CreateInstance(type))
               .ToList();
 
+        /// <summary>
+        ///   Locates the identity samples within the solution and creates an instance
+        ///   that can be inspected and run.
+        /// </summary>
+        ///
+        /// <returns>The set of samples defined in the solution.</returns>
+        ///
+        private static IReadOnlyList<IEventHubsIdentitySample> LocateIdentitySamples() =>
```

Comment:
You could make one of these generic and collapse to a single method.

Reply:
Nice one :)